modules/apps/object/object-test/src/testFunctional/tests/ObjectModelBuilder.testcase

```diff
@@ -1,6 +1,7 @@
 @component-name = "portal-object"
 definition {
 
+	property custom.properties = "feature.flag.LPS-148856=true";
```

Comment:
@davisaints this was already added yesterday. Can you do a rebase? Thanks!

Reply:
Done!